test-resources/ns_unordered.clj

```diff
@@ -0,0 +1,4 @@
+(ns ns-unordered
+  (:require
+   [clojure.string :as foo]
+   [clojure.stacktrace :as bar]))
```

Comment:
Alternative: use visual distinctive namespaces so you can spot the difference in one blink of an eye